unlock-app/package.json

```diff
@@ -18,7 +18,9 @@
     "connected-next-router": "^1.0.1",
     "cross-env": "^5.2.0",
     "ethereumjs-util": "^6.0.0",
+    "fs": "0.0.1-security",
```

Comment:
Isn't the `fs` from node.js good enough?

Reply:
YES and this may explain a lot ... I'm fighting with mocking this and I suspect this is why.